package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-es-logs",
-  "version": "3.0.3",
+  "version": "3.1.0",
```

Comment:
Can you revert this change to the version?

Reply:
Done